app/quest-boost/claim/[boostId]/page.tsx

```diff
@@ -198,16 +204,22 @@ export default function Page({ params }: BoostQuestPageProps) {
               )}
             </div>
 
-            {isUserWinner ? (
-              <div className={styles.claim_button_animation}>
+            <div className={styles.claim_button_animation}>
+              {isUserWinner ? (
                 <Button
                   disabled={(boost?.claimed && isUserWinner) || !isUserWinner}
                   onClick={handleClaimClick}
                 >
                   Collect my reward
                 </Button>
-              </div>
-            ) : null}
+              ) : (
+                <div className="block ml-auto mr-auto">
+                  <Button onClick={() => router.back()}>
```

Comment:
Sorry I should have said it the first time but it makes more sense to make the user go to `/quest-boost` instead of `.back()` because back makes him go to the quest boost he just opened